### PR TITLE
[MCC-112916] Implemented JsonSerializer

### PR DESCRIPTION
**Pull request by kenyamat**
@mdsol/hypermedia-team @edandersen Implemented JsonSerializer. Please review this.

### CODE REVIEW

File: src/Crichton.Representors/Serializers/JsonSerializer.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        {
+            if (representor == null)
+            {
+                throw new ArgumentNullException("representor");
+            }
+
+            return SerializeToJObject(representor).ToString();
+        }
+
+        protected JObject SerializeToJObject(CrichtonRepresentor representor)
```
Comment (edandersen) [inline src/Crichton.Representors/Serializers/JsonSerializer.cs]: SerializeAttributesToJObject

File: tests/Crichton.Representors.Tests/Serializers/JsonSerializerTests.cs [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+        }
+
+        [Test]
+        public void DeserializeToNewBuilder_SetsNestedAttribute()
+        {
+            var href = Fixture.Create<string>();
+            var json = @"
+            {
+                ""_links"": {
```
Comment (edandersen) [inline tests/Crichton.Representors.Tests/Serializers/JsonSerializerTests.cs]: Maybe change this from _links as the Json serializer does not support transitions

**Comment by kenyamat** [comment]
@edandersen Thank you for your feedback.
I fixed your feedback. Please review again.

**Comment by edandersen** [comment]
Perfect! Merging. Thank you.

**Pull request merged**